Misc/NEWS.d/next/Library/2017-07-05-14-48-26.bpo-30553.Oupsxo.rst

```diff
@@ -0,0 +1 @@
+Add HTTP/2 status code 421 (Misdirected Request) to http.HTTPStatus.
```

Comment:
Please add "Patch by Vitor Pereira.".

Reply:
Is this new? I sent a patch a couple of months ago and wasn't asked to do this.

Reply:
No, it's not new. We always try to add contributor's name in the NEWS file. Our GitHub workflow is still new so it's normal to forget this step (or sometimes we skip it intentionally if the patch is trivial)